server_tests/test/clientTests.js

```diff
@@ -1337,7 +1337,7 @@ describe("RUST API TEST SUITE", function() {
             return Utility.restoreDefaultOptions(defaultOptionsFile)
             .then(function() {
                 return Utility.deleteAllImages();
-            });
+            }, wrapError);
```

Comment:
Personally I like the latter approach everywhere, as it's more clear and clean; either works though, they both have the same effect.